src/tests/Interop/ICustomMarshaler/Primitives/ICustomMarshaler.cs

```diff
@@ -699,7 +718,11 @@ public static int Main(String[] args)
                 Parameter_CleanUpNativeDataMethodThrows_ThrowsActualException();
                 Field_ParentIsStruct_ThrowsTypeLoadException();
                 Parameter_DifferentCustomMarshalerType_MarshalsCorrectly();
-                DelegateParameter_MarshalerOnRefInt_ThrowsMarshalDirectiveException();
+                if (SupportsComInterop)
```

Comment:
This feels like the wrong name for this condition. Maybe `SupportsExceptionInterop` would be a better name?